app/web/template/check_your_lpa.gohtml

```diff
@@ -183,49 +183,13 @@
         {{ tr .App "attorneys" }}
       </h3>
 
-      {{ template "attorney-details" (attorneyDetails .Lpa.Attorneys "check" .App) }}
+      {{ template "attorney-details" (listAttorneys .Lpa.Attorneys .App .AttorneyDetailsPath .AttorneyAddressPath .RemoveAttorneyPath) }}
 
       <h3 class="govuk-heading-m govuk-!-margin-top-9 govuk-!-margin-bottom-2">
         {{ tr .App "replacementAttorney" }}
       </h3>
 
-      <dl class="govuk-summary-list">
-        <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
-          <dt class="govuk-summary-list__key">
-            {{ tr .App "name" }}
-          </dt>
-          <dd class="govuk-summary-list__value">
-            We haven't done this yet
-          </dd>
-        </div>
-
-        <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
-          <dt class="govuk-summary-list__key">
-            {{ tr .App "dateOfBirth" }}
-          </dt>
-          <dd class="govuk-summary-list__value">
-            We haven't done this yet
-          </dd>
-        </div>
-
-        <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
-          <dt class="govuk-summary-list__key">
-            {{ tr .App "email" }}
-          </dt>
-          <dd class="govuk-summary-list__value">
-            We haven't done this yet
-          </dd>
-        </div>
-
-        <div class="govuk-summary-list__row govuk-summary-list__row--no-actions">
-          <dt class="govuk-summary-list__key">
-            {{ tr .App "address" }}
-          </dt>
-          <dd class="govuk-summary-list__value">
-            We haven't done this yet
-          </dd>
-        </div>
-      </dl>
+      {{ template "attorney-details" (listAttorneys .Lpa.ReplacementAttorneys .App .ReplacementAttorneyDetailsPath .ReplacementAttorneyAddressPath .RemoveReplacementAttorneyPath) }}
```

Comment:
There's also the "Who is the replacement attorney" in section 1 that could be filled out now (although there are probably other changes needed like the how they work together bits here, so happy for you to defer)

Reply:
Yeah good point - we probably want a ticket for this page in the UR readiness epic. Will drop one in and handle there.